README.md

```diff
@@ -9,8 +9,8 @@ This is the source for the Gradle plugin for the
 [AssertJ Generator](http://joel-costigliola.github.io/assertj/assertj-assertions-generator.html). This plugin leverages
 existing Gradle [SourceSet](https://docs.gradle.org/current/dsl/org.gradle.api.tasks.SourceSet.html) API to make it
 easier to integrate the AssertJ Generator into existing Gradle-based projects. The configurations available mimic those
-provided by the [AssertJ Generator Maven Plugin](http://joel-costigliola.github.io/assertj/assertj-assertions-generator-maven-plugin.html).
-
+provided by
+the [AssertJ Generator Maven Plugin](http://joel-costigliola.github.io/assertj/assertj-assertions-generator-maven-plugin.html).
```

Comment:
I just realized we still host the generator documentation at the old website, I raised assertj/doc#139 to track this.